book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] In order to fully implement the has-many relationship (a book has many rentals), getters and setters for rentals should be present.

Reply:
Well solved